src/components/desktop/Header.js

```diff
@@ -1,14 +1,22 @@
+import { CaretDown } from '@styled-icons/boxicons-regular'
 import { User } from '@styled-icons/boxicons-solid'
 import { useTranslation } from 'react-i18next'
+import { NavLink, useLocation } from 'react-router-dom'
 import styled from 'styled-components/macro'
 
+const OnAbout = () => {
```

Comment:
Lets name this function using camelCase (e.g. `myFunction` not `MyFunction`)

Reply:
This is actually not how you should be using a hook (`useLocation`).

See the rules of hooks: https://reactjs.org/docs/hooks-rules.html

Instead, you can embed this inside the Dropdown component.